src/bufr_c2f_interface.F90

```diff
@@ -280,7 +285,7 @@ end subroutine mtinfo_c
     !> Wraps status() subroutine.
     !>
     !> @param file_unit - Fortran logical unit number of file.
-    !> @param lun - file ID.
+    !> @param lun - File ID.
     !> @param il - file status.
     !> @param im - message status.
```

Comment:
Would it make sense to capitalize 'file status' and 'message status' while we're at it?